config/api.yml

```diff
@@ -394,6 +394,7 @@
     - :collection
     :verbs: *gpppd
     :klass: MiqGroup
+    :parsed_by_attrs: description
```

Comment:
Couldn't come up with a better name (I'm all 👂 s).   

Semantically, this means "list of additional attributes other than id and href that this type of resource can be identified by".  

identifying_attrs maybe ? Other suggestions ?